src/components/ChartOfAccountsTable/ChartOfAccountsTable.tsx

```diff
@@ -257,6 +266,16 @@ export const ChartOfAccountsTableContent = ({
               >
                 View
               </Button>
+              {isDeletable && (
+                <Button
+                  variant={ButtonVariant.secondary}
+                  rightIcon={<Trash2 size={14} />}
+                  iconOnly
+                  onClick={onClickDelete}
+                >
+                  Delete
+                </Button>
+              )}
```

Comment:
I have a preference for always rendering the button even if it's not deletable, but if it's not, making the button disabled and adding a tooltip explaining that system accounts cannot be removed. Would rather users have some indication of why they can't delete things instead of just no button.